webpack.config.js

```diff
@@ -27,7 +27,6 @@ module.exports = {
   resolve: {
     extensions: [".ts", ".js", ".tsx", ".jsx"]
   },
-  externals: ["openmrs-config"],
```

Comment:
Wait is it `openmrs-config`? Or `module-config`? Are they in-browser modules in the import map?

Reply:
Still trying to figure out the name. Yes, it should be an in-browser module (well, JSON-as-module) in the import map. I think since I'm using the `System.import` API I don't need to list it as "external."